Formula/wownero.rb

```diff
@@ -0,0 +1,21 @@
+class Wownero < Formula
+  desc "Node and wallet binaries for the Wownero cryptocurrency project"
+  homepage "https://wownero.org"
+  url "https://git.wownero.com/attachments/56793f91-e5f1-4491-af96-516e53f6451d"
```

Comment:
Everything needs to be built from source.


https://docs.brew.sh/Acceptable-Formulae#we-dont-like-binary-formulae

Reply:
The Homebrew Cask repo makes me think it's only for GUI applications. Is that the case? I'd prefer to be lazy and just reference pre-built binaries.

Reply:
It is, or for applications that are not open source.

Reply:
Adjusted the formula to build from source.